internal/pkg/service/buffer/worker/service/slice_upload_test.go

```diff
@@ -1,734 +0,0 @@
-package service_test
-
-import (
-	"compress/gzip"
-	"context"
-	"io"
-	"strings"
-	"testing"
-	"time"
-
-	"github.com/benbjohnson/clock"
-	"github.com/keboola/go-client/pkg/keboola"
-	"github.com/keboola/go-utils/pkg/wildcards"
-	"github.com/stretchr/testify/assert"
-	etcd "go.etcd.io/etcd/client/v3"
-
-	"github.com/keboola/keboola-as-code/internal/pkg/idgenerator"
-	bufferDependencies "github.com/keboola/keboola-as-code/internal/pkg/service/buffer/dependencies"
-	"github.com/keboola/keboola-as-code/internal/pkg/service/buffer/store/model"
-	"github.com/keboola/keboola-as-code/internal/pkg/service/buffer/worker/service"
-	"github.com/keboola/keboola-as-code/internal/pkg/service/common/dependencies"
-	"github.com/keboola/keboola-as-code/internal/pkg/utils/errors"
-	"github.com/keboola/keboola-as-code/internal/pkg/utils/etcdhelper"
-	"github.com/keboola/keboola-as-code/internal/pkg/utils/strhelper"
-	"github.com/keboola/keboola-as-code/internal/pkg/utils/testhelper"
-	"github.com/keboola/keboola-as-code/internal/pkg/utils/testproject"
-)
-
-// TestSliceUploadTask - there are 2 slices, one is empty.
-// Both sliced are closed, but only the non-empty slice is uploaded to file storage.
-func TestSliceUploadTask(t *testing.T) {
```

Comment:
Je to pokryte:
https://github.com/keboola/keboola-as-code/blob/718568f24aaf898fb528a35b4aba9dfadd71741b/test/buffer/worker/worker_test.go#L25-L31

Obsah suboru nemusi kontrolovat, ked kontrolujeme obsah vyslednej tabulky po importe, tu:
https://github.com/keboola/keboola-as-code/blob/718568f24aaf898fb528a35b4aba9dfadd71741b/test/buffer/worker/003_file_import_test.go#L69-L99